Gemfile

```diff
@@ -1,13 +1,15 @@
 source 'https://rubygems.org'
 
 # Bundle edge Rails instead: gem 'rails', github: 'rails/rails'
-gem 'rails', '4.2.10'
+gem 'rails', '5.2.1'
+# Bootsnap for Rails 5 to optimize and cache expensive computations, Read more: https://github.com/Shopify/bootsnap
+gem 'bootsnap', '1.3.2'
 # Use SCSS for stylesheets
 gem 'sass-rails', '~> 5.0'
 # Use Uglifier as compressor for JavaScript assets
 gem 'uglifier', '>= 1.3.0'
 # Use CoffeeScript for .coffee assets and views
-gem 'coffee-rails', '~> 4.1.0'
+gem 'coffee-rails', '~> 4.2.2'
```

Comment:
Does Rails 5 ship with coffeescript still?

Reply:
[Yes](https://github.com/rails/rails/blob/5-2-stable/Gemfile#L23)

Reply:
Must be the not yet stable Rails 6.x that drops coffeescript support. Or maybe I just imagined them dropping support for it.